source/release-notes/1.5.txt

```diff
@@ -1,7 +1,7 @@
 .. _c2c-release-notes-1.5:
 
 ===================================
-Release Notes for mongosync 1.5
+Release Notes for ``mongosync`` 1.5
```

Comment:
[suggestion]

I think we ought to remove the backticks to be consistent with the [style guidelines](https://www.mongodb.com/docs/meta/style-guide/style/titles-and-headings/#style-and-structure): "Don't apply font treatments (bold, italics, or monospace) to text in a title or heading." This applies to all page titles.

Reply:
Fixed!